Makefile

```diff
@@ -0,0 +1,19 @@
+all: install_dependencies build_binaries format lint test 
+
+install_dependencies:
+	go get ./...
```

Comment:
`go mod tidy` helps with cleaning the mod files too, please check the difference